app/views/conferences/_about_and_happening_now.haml

```diff
@@ -16,9 +16,9 @@
   .container
     .row
       -# happening now events are displayed second in md or lg view
-      - if conference.splashpage.include_happening_now && conference.splashpage.include_program
+      - if conference.splashpage.include_happening_now? && conference.splashpage.include_program?
         - if conference.description.present?
-          .col-md-6.col-md-push-6.col-lg-4.col-lg-push-8
+          .col-md-6.col-md-push-6.col-lg-4.col-lg-push-8{ style: 'margin-top: 60px' }
```

Comment:
If extra styling is necessary, perhaps placing the styling into a `happening_now` CSS class would be best?

Reply:
so this is only relevant to this very instance of happening_now

Reply:
Hm ok then